client/src/Components/ResultBox.js

```diff
@@ -0,0 +1,9 @@
+import React, { useState } from 'react';
+
+
+const ResultBox = () => {
+    return(
+<input className="resultBox" type="text" id="handling" name="result"></input>
```

Comment:
The label is for "result", so the id of the Input field must be "result" as well